### PR TITLE
Covid App

### PR DESCRIPTION
**Pull request by sjdev2212**
In this project I :
Created an application using react-redux.
In this app, the user can check the number of cases, and deaths of covid in any South American country.
This functionality is achieved by consuming the media-group API by using redux and Axios.
I also did testing using react testing library.
Checked linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+## Built With
+
+- React and Redux
+- Redux-thunk, Redux-logger, axios.
+- React Testing Library
+- Covid 19 API
+
+## Live Demo
+
+[Live Demo Link](https://react-redux-capstone-sj1978.herokuapp.com//)
```
Comment (omar-labana) [inline README.md]: - [x] This link is broken, please remove the extra `/`.

File: src/components/Countries/Countries.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            Last Updated:
+            {element.All.updated}
+          </p>
+
+        </div>
+      ))}
+
+      <Link to="/">Back Home</Link>
+
+    </section>
```
Comment (omar-labana) [inline src/components/Countries/Countries.js]: - [x] The Home page should contain a filter (ideally with a controlled input component) that uses given parameters, e.g. category country name to search the list of countries.
- [x] The filtering should be done using a stateless component to filter data retrieved from the API.

File: src/App.js [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
-  <h1>
-    CAPSTONE
-  </h1>
-</section>
+    <section className="container wrapper">
+
+      <Routes>
+        <Route exact path="/" element={<Home />} />
+
+        <Route path="/countries" element={<Countries />} />
```
Comment (omar-labana) [inline src/App.js]: - [x] Each country should have its own unique route. I'd suggest taking a look at the `react-router-dom` dynamic routing.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @sj1978, 

Excellent work so far, you demonstrated your understanding of React and Redux!👏
The README contains instructions for installing and launching the project. ✔️
Video is professional and fulfills presentation requirements. 👌

There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [x] the problem with `dynamic require` is that since you aren't making use of dynamic routing you have to parse the local path to the required country image every time the component is rendered. By using dynamic routing `url/countryName`, you can access `URL parameters` via react-router then using a mapped hash you can call the required image file or preferably an external link to the required image (example {"Brazil":"/link to brazil flag/"}).

- [x] Kindly, cover the components with tests (I can't seem to find any). Also, please make use of the `React Testing Library` to mock the data fetching and Redux Store connection.

- [x] _Please, check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @sj1978, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**